### PR TITLE
add backers function and fix styling to project show

### PR DESCRIPTION
**Pull request by jdconrad89**
@Dpalazzari @drod1000 @kheppenstall @Robbie-Smith I added the backers function to the project model so we can call it in the views and fixed the styling of the project show page

### CODE REVIEW

File: app/views/shared/_projects.html.erb [hunk 1]
```diff
@@ -12,7 +12,7 @@
         <div>
           <h4><%= project.pledged %></h4>
           <p>pledged out of <%= project.goal %> </p>
-          <h3>1234</h3>
+          <h3><%= project.backers %></h3>
```
Comment (Dpalazzari) [inline app/views/shared/_projects.html.erb]: should this be `project.backers.count`?

Comment (jdconrad89) [inline app/views/shared/_projects.html.erb]: @Dpalazzari no sir i made a method in the projects controller... the actual people funding them if project.funders... this just returns the count so its something we don't have to call in the view

Comment (Dpalazzari) [inline app/views/shared/_projects.html.erb]: oh I literally just noticed. Hodor!

Comment (jdconrad89) [inline app/views/shared/_projects.html.erb]: hodor

**Pull request merged**